src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.targets

```diff
@@ -49,6 +49,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup Condition="'$(NuGetBuildTasksPackTargets)' == '' AND '$(ImportNuGetBuildTasksPackTargetsFromSdk)' != 'false'">
     <IncludeBuildOutput Condition=" '$(PackAsTool)' == 'true' ">false</IncludeBuildOutput>
     <PackageType Condition=" '$(PackAsTool)' == 'true' ">DotnetTool</PackageType>
+    <RuntimeIdentifiers Condition=" '$(PackAsTool)' == 'true' ">$(RuntimeIdentifiers);$(PackageToolShimRuntimeIdentifiers)</RuntimeIdentifiers>
```

Comment:
It looks wrong that this is in this property group. Why does the condition apply to it?

Reply:
It need to be set early enough before nuget is loaded. Only when PackAsTool=true aka use dotnet pack to create global tools this is useful. Also I want to add error when PackAsTool!=true and PackageToolShimRuntimeIdentifiers is set. According to what I talked to Kathleen

Reply:
Note I did not move this logic due to the reason above. It is out of date due to rename change